src/main/java/uk/gov/ons/census/casesvc/model/dto/RefusalDTO.java

```diff
@@ -10,8 +11,16 @@
 @JsonInclude(Include.NON_NULL)
 public class RefusalDTO {
   private RefusalTypeDTO type;
-  private String report;
   private String agentId;
   private String callId;
+  private boolean isHouseholder;
+
+  @JsonProperty("isHouseholder")
```

Comment:
this is not necessary

Reply:
There's no need to use the `@JsonProperty` annotation - you can remove it. It's not doing anything useful

Reply:
It sets the flag in the DB to the name isHouseholder rather than changing it to householder. If I remove it, the flag will always be false even if it is set to true in the event message. With it, it makes the difference.